app/models/grant_submission/submission_applicant.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module GrantSubmission
+  class SubmissionApplicant < ApplicationRecord
+    self.table_name = 'grant_submission_submission_applicants'
+    has_paper_trail versions: { class_name: 'PaperTrail::GrantSubmission::SubmissionApplicantVersion' },
+                    meta:     { grant_submission_submission_id: :grant_submission_submission_id,
+                                applicant_id: :applicant_id }
+
+    belongs_to :submission,   class_name: 'GrantSubmission::Submission',
+                              foreign_key: 'grant_submission_submission_id',
+                              inverse_of: :submission_applicants
+    belongs_to :applicant,    class_name: 'User',
+                              foreign_key: :applicant_id,
+                              inverse_of: :submission_applicants
+
+    # before_destroy :prevent_last_permission_destroy, if: -> { is_last_permission? }
+
+    validates :submission,  presence: true
+    validates :applicant,   presence: true
+
+    private
+
+    # def prevent_last_permission_destroy
+    #   errors.add(:base, 'There must be at least one permission on the submission')
+    #   throw :abort
+    # end
+
+    # def is_last_permission?
+    #   submission.permissions.one?
+    # end
```

Comment:
remove comments from other model

Reply:
In our meeting with Laura, we decided that this needs to exist. I think that in order to implement this functionality, we need to add discard to submission. Since this functionality is coming, I would rather keep the comments in.

Reply:
A couple of things here -- The general functionality is not dependent on discard. Not sure if we want to keep deleted submissions since they also likely come with attached files. As it stands, though, these validations could be added now.